docs/src/index.md

```diff
@@ -0,0 +1,77 @@
+# DustExtinction.jl
+
+```@contents
+```
+
+## Installation
+
+From the REPL, press `]` to enter Pkg mode
+```
+(v 1.1) pkg> add DustExtinction
+[...]
+```
+
+## Usage
+
+###Color laws
+
+```julia
```

Comment:
Use the `jldoctest` type, so that we can turn examples into actual tests and make sure they'll be updated.  See https://juliadocs.github.io/Documenter.jl/stable/man/doctests/  No need to do this in the previous block.